main.test.js

```diff
@@ -1,6 +1,13 @@
 const calculateWordsStatistic = require('./analysis.js');
+const readWords = require('./readfiles.js');
 
 test('check simple text statistic', () => {
-	const testText = "some test text, lala, test";
+	const testText = "some Test text, lala, test";
 	expect(calculateWordsStatistic(testText)).toEqual({ 'some': 1, 'test': 2, 'text': 1, 'lala': 1 });
-})
\ No newline at end of file
+});
+
+test('check simple text statistic', () => {
```

Comment:
This test should have own description instead of copied: `'check simple text statistic'`